server/honeybadger/processors.py

```diff
@@ -60,7 +61,8 @@ def process_wlan_survey(data):
     logger.info('Processing wireless survey data.')
     os = data['os']
     _data = data['data']
-    content = _data.decode('base64')
+    #content = _data.decode('base64')
+    content = b64d(_data).decode()
```

Comment:
Decoding again?

Reply:
Same issue as before. The base64 library also returns a bytes object. A pain, but somewhat understandable given how base64 operates. Again in a python shell, I can verify that data being base64 encoded needs to be encoded into a bytes object, and data being base64 decoded needs to be decoded into a string object after the fact.

The only reason that I did any encoding/decoding is that the script would throw type errors during my 2 to 3 conversions.

Reply:
I figured that was the case, but just wanted to make sure. Using requests will help with some of this, but the base64 thing is what it is. I'm surprised the python3 libs aren't operating in unicode by default though. I thought that was one of the big changes.